docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,65 +1,31 @@
 [id="planning_development_work"]
 = Planning your development work
 
-After creating a space, use the {osio} planner to plan your development work and to ensure efficient execution of your project.
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
 
-NOTE: The development process you select while creating the space determines the planner's guided work item type hierarchy.
+{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the planner's guided work item type hierarchy.
```

Comment:
@mishaone Re: Capitalization we agreed to use small case for planner/analytics instead of Title case as these are not exactly referenced in the UX. I remember us changing the title cases to small earlier after discussing this.

Reply:
Fixed the possessive.